stdlib/public/runtime/Casting.cpp

```diff
@@ -44,6 +44,11 @@
 #include <cstring>
 #include <type_traits>
 
+#if defined(__GLIBCXX__) && __GLIBCXX__ < 20160726
+#include <stddef.h>
```

Comment:
…and this could still be `cstddef`, right?

Reply:
`max_align_t` is defined by `stddef.h`, so this is safer.  I did have a look at the failure now that I was able to reproduce it, and this definitely fixed it.